auth/cache/memory_test.go

```
@@ -0,0 +1,132 @@
+
+	incremented, err := driver.IncrementFUPEntry("key")
+	if nil != err {
+		t.Fatalf("IncrementFUPEntry() error = %v", err)
+	}
+	incremented.Used[constants.PeriodDaily] = 42
+
+	stored, err := driver.GetFUPEntry("key")
+	if nil != err {
+		t.Fatalf("GetFUPEntry() error = %v", err)
+	}
+	if got := stored.GetUsed(constants.PeriodDaily); 1 != got {
```

golangci-lint (2.12.2)

[error] ST1017: don't use Yoda conditions

(staticcheck)